bedrock/firefox/templates/firefox/pocket.html

```diff
@@ -0,0 +1,116 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-resp.html" %}
+
+{% block page_title %}{{ _('Pocket') }}{% endblock %}
+{% block body_id %}firefox-pocket{% endblock %}
+{% block body_class %}
+  {{ super() }}
+  mozid
+{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_pocket' %}
+{% endblock %}
+
+{% block tabzilla_css %}{% endblock %}
+{% block tabzilla_js %}{% endblock %}
+
+{% block site_header %}
+    {% block tabzilla_tab %}
+    <div class="content">
+      <div id="tabzilla">
+        <a href="{{ url('mozorg.home') }}">Mozilla</a>
+      </div>
+    </div>
+    {% endblock %}
+    {% block site_header_nav %}{% endblock %}
+    {% block site_header_logo %}{% endblock %}
+    {% block breadcrumbs %}{% endblock %}
+{% endblock %}
+
+{% block content %}
+<main role="main">
+  <section class="content-main">
+    <div class="content">
+      <h1>{{ _('Pocket for Firefox') }}</h1>
+
+      <h2>
+        {{ _('Save it now. Read or watch it anytime, anywhere.') }}
+        <br>
+
+        {# Current Firefox #}
+        <span class="has-firefox">{{ _('All you need is your Firefox Account.') }}</span>
+
+        {# Old Firefox or Not Firefox #}
+        <span class="get-firefox">{{ _('Get Firefox to start using Pocket right in your browser.') }}</span>
+      </h2>
+
+      {# Has toolbar button #}
+      <p class="main-cta" id="cta-try"><button type="button" class="button-flat try-pocket">{{ _('Try it now') }}</button></p>
+
+      {# No toolbar button #}
+      <p class="main-cta" id="cta-learn"><a href="https://support.mozilla.org/kb/where-pocket-button-firefox" class="button-flat">{{ _('Learn more') }}</a></p>
+
+      {# Old Firefox or Not Firefox #}
+      <p class="main-cta" id="cta-download"><a href="{{ url('firefox.new') }}?scene=2#download-fx" class="button-flat-green">{{ _('Download Firefox') }}</a></p>
```

Comment:
After much discussion on IRC, we need the button to say "Download Firefox" to be clear on what the person is downloading (they're not downloading Pocket), so this needs to be a link to /new. And we'll link directly to scene 2 because we lose a lot of potential downloads when folks have to click an extra time, while scene 2 fires the download automatically. This doesn't work in older IEs but we're willing to accept that loss.